RetrospectiveExtension.Frontend/dal/processService.tsx

```diff
@@ -77,13 +80,13 @@ export class ProcessService {
     }
 
     public getProcessForCurrentProject = () => {
-        const currentProjectId = VSS.getWebContext().project.id;
-
-        return this.getProcesses().then(processModels => {
-            return processModels.find((process) => {
-                const projects = process.projects ? process.projects : [];
-                return projects.some((project) => project.id === currentProjectId);
-            });       
+        return getProjectId().then(currentProjectId => {
```

Comment:
method is great candidate for async/ await refactor if the context allows it

Reply:
The old code actually uses `.then` so when I changed it, I want to follow the same pattern. If we want to switch to async, we should let @polatengin decide that and do it for the whole file for consistency.